### ISSUE TITLE
Wheat TD file format issue

### ISSUE THREAD

**Issue by ShresthaR** [Planteome/CO_321-wheat-traits#10]
Hi @marieALaporte, 
Could you please provide me the latest version of the wheat TD in Excel format? I downloaded the wheat_TD.csv from here (github) which is available under the "Code" tab but when I tried to open it, the file was screwed up. We can use "semi-colon" as a delimiter in Excel, and it looks better but since "comma" has been used in many columns to separate authors, or in descriptions or in Reference columns, the changing into .csv file causing this issue I guess. So, when we use this option, is does not work properly and cannot see information correctly in designated columns/headers. I could not find an easy way to formatting it. The TD that we can download from the CO website has also same problem. It would be easier for the curators if they can just download Excel.   
Many thanks

**Comment by ShresthaR** [comment]
I removed the commas from the wheat_TD.csv and now with the semi-colon delimiter, it is workable.

**Comment by cooperl09** [comment]
I was able to open the wheat_TD.csv in both Excel and Google sheets, with no issues.  Just select the semicolon as the delimiter.

**Comment by kadreher** [comment]
Greetings @marieALaporte , @cooperl09 , and @ShresthaR . I would like to add my request to please provide the trait dictionary in an easily accessible format, e.g. an Excel file available for download and immediate use. I understand that it is not ideal to use proprietary formats but I think that many people can access Excel or Google sheets to use an .xlsx file.  At least for me, the .csv file extension is problematic because applications like Excel then assume that it is comma-delimited. I could only open it successfully without getting distorted and separated information if I first re-saved it with .txt extension and then tried to re-open it in Excel where I could then choose a different delimiter.
However, I think that it would be wonderful if we could save others from having to do these manipulations by providing an Excel file for download from this site or the Crop Ontology site or both.

**Comment by elizabetharnaud** [comment]
@kadreher Hi Kate. Did you tried to directly open the csv in Excel with 'File', 'open' ? I usually do that and there is no r'esulting problem of format. If you use the 'import' , 'csv' file then  it is a problem indeed. Let me know

**Comment by kadreher** [comment]
Hi @elizabetharnaud . Thank you for checking. When I download the csv file from the Crop Ontology site or try to save it from Github and then just try to open in Excel, unfortunately, it appears in a format that I cannot use. Here is an image:
![image](https://user-images.githubusercontent.com/8202843/94595950-7698c380-0259-11eb-9624-bfac7a6a592c.png)

**Comment by cooperl09** [comment]
Here is an easy way to open the wheat_TD.csv file in Excel:
1. Open a new blank spreadsheet in Excel
2. Under File > import > csv file.  Select the wheat_TD.csv file on your local computer
3. Select 'Delimited', if not already selected > click 'Next'
4. Select Semicolon as the delimiter.  Uncheck 'Tab'
5. Leave Column data format 'General' on next screen
6. Click Finish
 Sheets displays fine:
![image](https://user-images.githubusercontent.com/7378000/94606590-8e6b4a00-024f-11eb-968d-f15ca1657087.png)

**Comment by cpommier** [comment]
If this method works for everybody, it could be added in a README.md alongside the wheat_TD.csv. We could even include a "how to contribute" section to clarify the workflow. What do you think @elizabetharnaud ? I can create a pull request with that one if you want.

**Comment by elizabetharnaud** [comment]
Yes. I was thinking even to add these explantions to the web site directly . We are currently revising the Guidelines for developing a TD  and i wish to have specific guidelines for the submission workflow that appears a bit complexe to the curators .
